libcontainer/configs/cgroup_unix.go

```diff
@@ -16,6 +16,9 @@ type Cgroup struct {
 	// name of parent cgroup or slice
 	Parent string `json:"parent"`
 
+	// CgroupsPath specifies the path to cgroups that are created and/or joined by the container.
+	CgroupsPath string `json:"cgroupsPath"`
```

Comment:
Yes, that's what I planed, when we merge this, we can start removing `Parent` and `Name`.